platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch

```diff
@@ -160,7 +160,7 @@ index 979352299..3771d19c9 100644
 +                    break;
```

Comment:
These lines (160-162) maybe from a different branch, please validate this is required for rebranding

Reply:
checked! ,the same!